src/process/id.rs

```diff
@@ -267,6 +267,20 @@ pub fn getppid() -> Option<Pid> {
     backend::process::syscalls::getppid()
 }
 
+/// `getpgid()`—Returns the process group ID of the given process.
```

Comment:
There should be a `pid` inside the parens there. The current convention is "pseudo-code for what this looks like in C".